packages/components/aside/src/aside.js

```diff
@@ -68,6 +71,13 @@ customElementDefineHelper(
 			`;
 		}
 
+		handleSlotPlatformObjectChange(e){
+			const slot = e.currentTarget;
+			const assignedNodes = slot.assignedNodes();
+			const showSlot = assignedNodes && assignedNodes.length;
+			slot.parentElement.classList[showSlot ? 'remove' : 'add']('hidden');
```

Comment:
Can you just set an internal property (same as hasFoot) here and then do the conditional class name in the 
```
<div class="aside-platform-object ${this.hasPlatformObject ? '' : 'hidden'}">
```

Reply:
Done. Any reason why it is preferred to classList approach (which I took from `root.js`), is it a question of elegance? Also should we not have the same for footer? If the footer is removed (because the same aside element is used to show different aside, one with footer and the other without?) it will still be displayed.

Reply:
As you can see in `root.js` file the `decorateSlot` was used in lots of slots, so it tries to don't duplicate the code! 
The `defaultSlotChangeHandler` is a bit different and can be approached in both way but the difference here is you have a default class name in place which is better to just show the logic in there to make it easier to follow